ios/Video/ReactNativeVideoManager.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ReactNativeVideoManager.swift
+//  react-native-video
+//
+
+import Foundation
+
+public class ReactNativeVideoManager: RNVAnalyticsPlugin {
+    private let expectedMaxVideoCount = 10
+
+    // create a private initializer
+    private init() {}
+
+    public static let shared: ReactNativeVideoManager = {
+        let instance = ReactNativeVideoManager()
+        // setup code
+        return instance
+    }()
```

Comment:
We can simplify this
```suggestion
    public static let shared: ReactNativeVideoManager = ReactNativeVideoManager()
```
(Btw if you look bellow there are comments for "ReactExoplayerViewManager" on iOS 😉)

Reply:
Done thank you